redis/src/main/java/org/apache/zeppelin/redis/RedisInterpreter.java

```diff
@@ -0,0 +1,98 @@
+package org.apache.zeppelin.redis;
+
+import org.apache.zeppelin.interpreter.Interpreter;
+import org.apache.zeppelin.interpreter.InterpreterContext;
+import org.apache.zeppelin.interpreter.InterpreterResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import redis.clients.jedis.Jedis;
+
+import java.util.HashSet;
+import java.util.Properties;
+import java.util.Set;
+
+import static org.apache.zeppelin.interpreter.InterpreterResult.Code.ERROR;
+import static org.apache.zeppelin.interpreter.InterpreterResult.Code.SUCCESS;
+
+public class RedisInterpreter extends Interpreter {
+
+    private Logger LOG = LoggerFactory.getLogger(RedisInterpreter.class);
+
+    private Jedis client;
+
+    private enum RedisCommand {
+        GET,SET
+    }
+
+    private Set<String> keywords = new HashSet<>();
+
+    public RedisInterpreter(Properties property) {
+        super(property);
+
+        for (RedisCommand command : RedisCommand.values()) {
+            keywords.add(command.name());
+        }
+    }
+
+    @Override
+    public void open() {
+        client = new Jedis();
```

Comment:
I read Jedis source code and find a strange case . The `returnResource` will be deprecated after `3.0` and the resource shoule be close after usage (don't return into source pool ) .  Now I'm using `2.9` . I'm not sure it's a good idea .